### PR TITLE
Fixes #9131: Add capabilities in inventory

### PR DESCRIPTION
**Pull request by Kegeruneku**
https://www.rudder-project.org/redmine/issues/9131

### CODE REVIEW

File: rudder-agent/SOURCES/patches/fusioninventory/0018-Fixes-9131-add-agent-capabilities-to-rudder-inventory.patch [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+     return @server_roles;
+ }
+ 
++sub _listAgentCapabilities {
++    my $capabilities_file = ($OSNAME eq 'MSWin32') ? 'C:\Program Files\Rudder\etc\agent-capabilities' : '/opt/rudder/etc/agent-capabilities';
++    my @capabilities;
++
++    # List agent capabilities, one per line in the file
++    if (-f "$capabilities_file") {
++        if (open(my $fh, '<:encoding(UTF-8)', $capabilities_file)) {
```
Comment (peckpeck) [inline rudder-agent/SOURCES/patches/fusioninventory/0018-Fixes-9131-add-agent-capabilities-to-rudder-inventory.patch]: encoding(UTF-8), not needed but cool :)

File: rudder-agent/SOURCES/patches/fusioninventory/0018-Fixes-9131-add-agent-capabilities-to-rudder-inventory.patch [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
++            while (my $row = <$fh>) {
++                chomp $row;
++                push @capabilities, $row;
++            }
++        }
```
Comment (peckpeck) [inline rudder-agent/SOURCES/patches/fusioninventory/0018-Fixes-9131-add-agent-capabilities-to-rudder-inventory.patch]: Even if this is a single script, you should close the files you open -> close $fh

**Comment by Kegeruneku** [comment]
Commit modified

**Comment by peckpeck** [comment]
Good

**Pull request merged**